nb_train.py

```diff
@@ -4,87 +4,159 @@
 import nltk
 import random
 import pickle
-from sklearn.model_selection import train_test_split
 from nltk.corpus import stopwords
 from zipfile import ZipFile
 
-with ZipFile('imdb ratings.zip') as zf:
-    f = zf.open('movie.csv')
-    
-df = pd.read_csv(f)
-smalldf = df[:2000]
-smalldf = smalldf.drop_duplicates(ignore_index = True)
-smalldf.loc[smalldf['label'] == 0, 'label'] = 'neg'
-smalldf.loc[smalldf['label'] == 1, 'label'] = 'pos'
+DATA = 'movie.csv'
+FEATURE_0 = 'text'
+LABEL = 'label'
+LABEL_0 = 0
```

Comment:
Here's a good case in point... `LABEL_0` isn't any better than `0`. It doesn't inform the reader of the code of the *meaning* of the thing in question.

Instead: `NEGATIVE_SENTIMENT_LABEL`